core_backend/tests/api/conftest.py

```diff
@@ -197,15 +198,17 @@ def fullaccess_token() -> str:
     """
     Returns a token with full access
     """
-    return create_access_token("fullaccess")
+    return create_access_token("user1")  # TEMPORARY HARDCODED USER ID
 
 
 @pytest.fixture(scope="session")
 def readonly_token() -> str:
     """
     Returns a token with readonly access
     """
-    return create_access_token("readonly")
+    return create_access_token(
+        "user1"
+    )  # TEMPORARY HARDCODED USER ID - this is also fullaccess!
```

Comment:
Not sure about this - how does it know to return a readonly token?

Reply:
The term "readonly" is baked into the token. So when it is unencrypted, we retrieve that user's access level from `USERS` global. See `get_current_user`.

Reply:
Oh I seee, this is the JWT stuff. Nice.